idx_width: 55
idx_middle: 388
height_idx_low: -15
height_idx_up: -5
min_ang_idx: 38

laser_offset_x: 0.0
laser_offset_y: 0.0
laser_offset_z: 0.0
 
occ_res: 0.1
cloud_res: 0.08
shift_yaw: -1.5708

min_theta: -3.927

max_angular_correction_deg: 2.5
max_linear_correction: 0.5
max_iterations: 5
epsilon_xy: 0.001
epsilon_theta: 0.01
max_correspondence_dist: 1.0
do_compute_covariance: 0
